domains/ciaobot.json

```diff
@@ -0,0 +1,12 @@
+{
+    "owner": {
+        "username": "Ciao287",
+        "email": "",
+        "discord": "687333016921440317"
+    },
+    "record": {
+        "A": [
+            "84.46.245.191"
```

Comment:
Unable to access site.

Reply:
84.46.245.191 is the ip of the reverse proxy server of the host i use, when the domain is merged the owner of the host will link the url to my site, my current url is http://server.nexcord.com:11102/